Pipfile

```diff
@@ -7,8 +7,9 @@ verify_ssl = true
 pbr = "*"
 
 [packages]
-rdflib = "~=5.0"
-rdflib-jsonld = "~=0.5"
```

Comment:
it looks like you are getting rid of this, but is it not required if only rdflib 5.0 is used?

Contrast with #79 which mirrors the changes in linkml